NEWS.md

```diff
@@ -1,3 +1,58 @@
+flux-core version 0.48.0 - 2023-03-07
+-------------------------------------
+
+This release adds submission directives ("see flux help batch") and
+shortens the the job submission commands to "flux batch", "flux run",
+etc..  The flux-mini(1) command is deprecated.
```

Comment:
Super minor nit: I _think_ when "etc." is the last word in a sentence, you do not need a second period, the period at the end of "etc" is sufficient. 😅

Reply:
You are right!  Great catch!

https://english.stackexchange.com/questions/8382/when-etc-is-at-the-end-of-a-phrase-do-you-place-a-period-after-it

I believe I am a repeat offender:
```console
$ cd doc
$ git grep 'etc\.\.'
man3/flux_event_decode.rst:   missing topic string, etc..
man3/flux_request_decode.rst:   missing topic string, etc..
man3/flux_response_decode.rst:   missing topic string, etc..
```

Reply:
Just pushed fix.  I'll take care of those others in #4934